SPECS/tensorflow/tensorflow.signatures.json

```diff
@@ -0,0 +1,6 @@
+{
+    "Signatures": {
+     "tensorflow-2.8.3.tar.gz": "4b7ecbe50b36887e1615bc2a582cb86df1250004d8bb540e18336d539803b5a7",
+     "tensorflow-2.8.3-cache.tar.gz":"700bec907a2f762adf0ca5b65112dfd0e4e3e26cdaa4a02de024131b8f1114e0"
```

Comment:
how this tarball is generated ?

Reply:
if not downloaded from upstream, could you add a script that builds it (see example in mariadb)